src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,227 @@
+        return description.contains(keyword);
+    }
+
+    /**
+     * Lists all the tasks in the TaskList.
+     * @param tasks An ArrayList of Tasks
+     */
+    public static void listTasks(ArrayList<Task> tasks) {
+        System.out.println("----------------------------------------------");
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < tasks.size(); i++) {
+            System.out.println((i + 1) + ". " + tasks.get(i));
+        }
+        System.out.println("----------------------------------------------");
+    }
+
+    /**
+     * Marks a task in the TaskList as "completed".
+     * @param tasks An ArrayList of Tasks
+     * @param index Index of task in ArrayList
+     * @throws IndexOutOfBoundsException If user provides a task number
+     * that does not exist
+     */
+    public static void markTaskDone(ArrayList<Task> tasks, int index)
+            throws IndexOutOfBoundsException {
+        System.out.println("----------------------------------------------");
+        Task t = tasks.get(index - 1);
+        t.markAsDone();
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println(t);
+        System.out.println("----------------------------------------------");
+    }
+
+    /**
+     * Removes a task from the TaskList.
+     * @param tasks An ArrayList of Tasks
+     * @param index Index of task in ArrayList
+     */
+    public static void deleteTask(ArrayList<Task> tasks, int index) {
+        System.out.println("----------------------------------------------");
+        Task t = tasks.remove(index - 1);
+        System.out.println("Noted. I've removed this task:");
+        System.out.println(t);
+        System.out.println("Now there are " + tasks.size() + " tasks on the list");
+        System.out.println("----------------------------------------------");
+    }
+
+    /**
+     * Creates a ToDoTask.
+     * @param tasks An ArrayList of Tasks
+     * @param command A String array containing the full command
+     */
+    public static void createTodo(ArrayList<Task> tasks, String[] command) {
+        System.out.println("----------------------------------------------");
+        String todo = "";
+        for (int i = 1; i < command.length; i++) {
+            todo += " " + command[i];
+        }
+        ToDoTask newTodo = new ToDoTask(todo, false);
+        tasks.add(newTodo);
+        System.out.println("Added this:");
+        System.out.println(newTodo);
+        System.out.println("Now there are " + tasks.size() + " tasks on the list");
+        System.out.println("----------------------------------------------");
+    }
+
+    /**
+     * Creates a DeadlineTask.
+     * @param tasks An ArrayList of Tasks
+     * @param command A String array containing the full command
+     */
+    public static void createDeadline(ArrayList<Task> tasks,
+                                      String[] command) {
```

Comment:
Are these breaks needed? I've seen them elsewhere as well. If you are not exceeding the hard 120 character limit maybe these could be in a single line to be a bit more readable? Also the next line should be 8 spaces from the original if you intend to break up a long line.